pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Related issues
```

Comment:
Is this the template for our internal PRs as well?
If yes: see trello ticket description
If no: nevermind me

Reply:
Nope, for internal PRs I'll create separate templates in every repo